source/documentation/using_the_api/datatypes.md

```diff
@@ -0,0 +1,26 @@
+## Datatypes
```

Comment:
I think this would be better if it were centred around a 'How to', as the CURIEs and foreign keys sections are, but the reason I've written it this way is to accommodate GOV.UK style limitations about numbered lists. I'm not sure if the focus should be on resolving foreign keys and CURIEs, though, in which case this should be more of a 'Prerequisites' section.

Reply:
I called this 'Datatypes' because it was much more straightforward to make the content GOV.UK-appropriate and to fit it in with the rest of the documentation rather than having it be focused on CURIEs and foreign keys and have ascertaining the domain/datatype be a prerequisite. If this approach doesn't make sense just let me know.